modules/BuildTools.Tasks/BuildTools.Tasks.csproj

```diff
@@ -9,16 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\**\*.cs" />
```

Comment:
The 2.0.0 version of this package exists on aspnetcore-master (which isn't afflicted by the rooster and scrambled omelette problem).